packages/components/src/pages/Quickstart.vue

```diff
@@ -0,0 +1,829 @@
+      <section class="qs-step" v-if="currentStep === 4">
+        <div class="qs-step__head">
+          <h1 class="qs-title">Open AppMaps</h1>
+        </div>
+        <div class="qs-step__block">
+          <p>We’ve identified a few AppMaps you may want to check out first.</p>
+          <p v-for="appmap in appmaps" :key="appmap.path">
+            <a href="#" @click="openAppmap(appmap.path)">{{ appmap.name }}</a>
+          </p>
+          <p>
+            Browse the
+            <a href="#" @click="openLocalAppmaps">local AppMaps</a> and open any
+            from the list. For more information, go
+            <a href="https://appland.com/docs/reference/vscode">here</a>
+          </p>
+        </div>
+      </section>
+    </div>
+    <div class="qs-help">
+      <HelpIcon class="qs-help__icon" />
+      <div class="qs-help__text">
+        <p>Need help or want to give feedback?</p>
+        <a
+          href="https://appland.com/company/contact-us"
+          target="_blank"
+          rel="noopener noreferrer"
+          >Contact an AppLand developer directly.</a
+        >
+      </div>
+    </div>
+    <div class="qs-popup-bg" v-if="showProjectSelector">
+      <article class="qs-popup">
+        <h1 class="qs-popup__title">Select a project</h1>
+        <p class="qs-popup__subtitle">
+          This workspace contains multiple projects. Select the project you
+          would like to map with AppMap
+        </p>
+        <div class="qs-popup__radio-group">
+          <div
+            class="qs-popup__radio"
+            v-for="project in projects"
+            :key="project.name"
+          >
+            <label class="qs-radio">
+              <input
+                type="radio"
+                :value="project.name"
+                v-model="selectedProject"
+              />
+              <span>{{ project.name }}</span>
+            </label>
+          </div>
+        </div>
+        <button class="qs-button" @click="saveSelectedProject">
+          Confirm project selection
+        </button>
+      </article>
+    </div>
+  </div>
+</template>
+
+<script>
+import HelpIcon from '@/assets/quickstart/help.svg';
+import StepLeftIcon from '@/assets/quickstart/arrow-left.svg';
+import StepRightIcon from '@/assets/quickstart/arrow-right.svg';
+import SuccessIcon from '@/assets/quickstart/success.svg';
+
+export const Steps = {
+  INSTALL_AGENT: 1,
+  CONFIGURE_APPMAP: 2,
+  RECORD_APPMAPS: 3,
+  OPEN_APPMAPS: 4,
+};
+
+export default {
+  name: 'Quickstart',
+
+  components: {
+    HelpIcon,
+    StepLeftIcon,
+    StepRightIcon,
+    SuccessIcon,
+  },
+
+  props: {
+    language: {
+      type: String,
+      default: null,
+    },
+    testFrameworks: {
+      type: Array,
+      default: () => [],
+    },
+    installSnippets: {
+      type: Object,
+      default: () => {},
+    },
+    appmapYmlSnippet: {
+      type: String,
+      default: '',
+    },
+    runTestsSnippet: {
+      type: String,
+      default: '',
+    },
```

Comment:
The snippet will vary for each framework sent along in the `testFrameworks` prop. Can we map test snippets to language? Otherwise the display logic for the snippet UI becomes the responsibility of the quickstart owner (the extension plugin).

Reply:
e.g.
```js
{
  minitest: 'APPMAP=true bundle exec rails test',
  rspec: 'APPMAP=true bundle exec rake',
}
```